frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnField.tsx

```diff
@@ -37,11 +38,14 @@ const MultiColumnField: React.FC<MultiColumnFieldProps> = ({
                 name={name}
                 rowIndex={index}
                 onDelete={() => remove(index)}
+                isReadOnly={isReadOnly}
               >
                 {children}
               </MultiColumnFieldRow>
             ))}
-          <MultiColumnFieldFooter addLabel={addLabel} onAdd={() => push(emptyValues)} />
+          {isReadOnly && (
```

Comment:
Did you mean:
```suggestion
          {!isReadOnly && (
```

Reply:
Yeah, it makes more sense now I am passing ```!pipelineParameterAccess``` from the parent so it will be ```!isReadOnly``` here.